components/ledger/pkg/ledger/volume_agg.go

```diff
@@ -66,6 +67,13 @@ func (tva *TxVolumeAggregator) Transfer(
 	tva.PostCommitVolumes.AddOutput(from, asset, amount)
 	tva.PostCommitVolumes.AddInput(to, asset, amount)
 
+	for account, volumes := range tva.PostCommitVolumes {
```

Comment:
You moved this code from the main loop to this "Transfer" function.
But this "Transfer" function is called multiple times.
Unless i'm wrong, doing this will introduce unnecessery processing.

Reply:
You're totally right... I'm reverting that, sorry.